app/views/reports/itt_new_starter_data_sign_off.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= render PageTitle::View.new(text: "Reports") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(
+    text: t("back"),
+    href: root_path,
+    ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+  <h1 class="govuk-heading-l">Export new trainee data for sign off for the <%= AcademicCycle.current.label %> academic year</h1>
+    <p class="govuk-body">This export will show the new trainee data you’ve provided to the DfE for trainees who’ve started their initial teacher training (ITT) in the <%= AcademicCycle.current.label %> academic year, and have a trainee start date on or before the second Wednesday of October <%= AcademicCycle.current.start_year %> (sometimes called the census date).</p>
+    <div class="govuk-inset-text">For the <%= AcademicCycle.current.label %> academic year, the ITT census date is <%= @census_date %>.</div>
+    <h2 class="govuk-heading-m">What you must do to sign off your new trainee data</h2>
+    <p class="govuk-body">Use this export to check you’ve provided accurate data for all your new trainees. You should check for any errors or missing trainees.</p>
+    <p class="govuk-body">Once you’ve checked your data, a senior person from your organisation (preferably a different person to who submitted the data) should <a href="https://forms.gle/qhYG4hMJT8jG9LEd6" class="govuk-link">sign it off using this form</a>.</p>
```

Comment:
Can this link please force a new tab? Syntax [here](https://design-system.service.gov.uk/styles/typography/#opening-links-in-a-new-tab).

Reply:
done